scripts/update-sources.ts

```diff
@@ -316,37 +319,34 @@ async function buildIosZano(platform: IosPlatform): Promise<void> {
     'install'
   ])
 
-  // Explode Zano archives and gather the objects:
-  async function unpackLib(libPath: string, name: string): Promise<void> {
-    console.log(`Unpacking lib${name}.a`)
-    const outPath = join(working, `unpack/${name}`)
-    await rm(outPath, { recursive: true, force: true })
-    await mkdir(outPath, { recursive: true })
-    await loudExec('ar', ['-x', libPath], { cwd: outPath })
+  // Link everything together into a single giant .o file:
+  console.log(`Linking zano-module.o for ${sdk} ${arch}`)
+  const objectPath = join(working, 'zano-module.o')
+  await loudExec(ld, [
```

Comment:
Wow. I wasn't aware of the `ld` command. This seems like the silver bullet to solve a lot of our conflicts.

Reply:
Yes, but only when combined with the `llvm-objcopy` it seems. This appears to be the insight. @swansontec correct me if I'm wrong.

Reply:
Right, merging the object files doesn't solve anything because the symbols are still there. We need a second tool to actually mark them as private so they won't confuse Xcode anymore.